src/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do List</title>
+</head>
+<body>
+  <main>
+    <section id="app">
+      <h2 id="title"><span>Today's To Do</span> 
+        <button type="button" id="refresh">
+          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="#bebebe" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M21.5 2v6h-6M2.5 22v-6h6M2 11.5a10 10 0 0 1 18.8-4.3M22 12.5a10 10 0 0 1-18.8 4.2"/></svg>
+        </button>
+      </h2>
+      <ul id="to-do-list"></ul>
```

Comment:
- Hey, from the image I added in the previous review, your design is missing the `clear completed` task button. The button is meant to be in the app so users can clear the list of tasks they have completed which is the subsequent project milestone, you will implement this functionality. please, kindly add the missing button.